test/e2e/e2e.go

```diff
@@ -43,7 +44,11 @@ const (
 	DefaultTestNamespace = "e2etest-knative-eventing"
 
 	interval = 1 * time.Second
-	timeout  = 1 * time.Minute
+	timeout  = 2 * time.Minute
```

Comment:
do we really need to increase the timeout here? 2 minutes seems really long? 🤔

Reply:
When I executed the presubmit-tests, the `WaitForLogContentCount` function got timeout error and after I changed the timeout to 2 minutes the error disappeared. 
Yay I also think 2 minutes is too long, I will investigate if it can be restored to 1 minute.

Reply:
Changed back to 1 minute and it worked :)